src/config/services/servicesConfig.go

```diff
@@ -132,6 +134,7 @@ func DecodeConfig(cfg config.Provider) (c Config, err error) {
 	if c.AASA.Prefix == "" {
 		c.AASA.Prefix = cfg.GetString(aasaPrefixKey)
 		c.AASA.Bundle = cfg.GetString(aasaBundleKey)
+		c.AASA.Version = cfg.GetString(aasaVersion)
```

Comment:
This line may need to be in its own if block but I'm not sure. I'd say, as long as we have test cases covering multiple scenarios and they pass, we're probably fine to leave this here.

Reply:
I moved this to it's own block